pkg/alertmanager/alertstore/local/store.go

```diff
@@ -148,9 +149,40 @@ func (f *Store) reloadConfigs() (map[string]alertspb.AlertConfigDesc, error) {
 		// The file name must correspond to the user tenant ID
 		user := strings.TrimSuffix(info.Name(), ext)
 
+		// Load template files
+		userTemplateDir := filepath.Join(f.cfg.Path, user, templatesDir)
+		var templates []*alertspb.TemplateDesc
+
+		if _, e := os.Stat(userTemplateDir); e == nil {
+			err = filepath.Walk(filepath.Join(f.cfg.Path, user, templatesDir), func(path string, info os.FileInfo, err error) error {
```

Comment:
we can replace `filepath.Join(f.cfg.Path, user, templatesDir)` with just `userTtemplateDir` variable right?